<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummaryTool</class>
 <widget class="QMainWindow" name="SummaryTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>622</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Summary Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/collection_placeholder.png</normaloff>icons/collection_placeholder.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Collection Panel</string>
     </attribute>
     <widget class="QPushButton" name="applyQueryButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>530</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Generate Results</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>369</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Summary Options</string>
      </property>
      <widget class="QRadioButton" name="radioReduced">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reduced Document</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioFull">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Full Document</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinSentencesCollection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string># Sentences</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinWordsCollection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string># Key Words</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="collectionBrowseButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>500</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>LSA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lex Rank</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>More coming soon!!</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="queryBrowseButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="collectionPathLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Path to Collection:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>500</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Summary Engine:</string>
      </property>
     </widget>
     <widget class="QLabel" name="progressLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="collectionPathInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="plainText">
       <string>C:\Users\zwelz3\Documents\GTRI_Projects\ECCT_EW_EMS\Market Research Sample</string>
      </property>
      <property name="placeholderText">
       <string>Enter location of collection root...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="processButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
     <widget class="QLabel" name="queriesComboLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Query Selection:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="queryLoadButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Query Options</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update Query File</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sentence Buffer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>false</underline>
       </font>
      </property>
      <property name="title">
       <string>Flags</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>341</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>cs - case sensitive,  cn - constituents</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>w - whole word,  p - plurals,  d - special delimeters</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="queriesCombo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="queryPathInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="placeholderText">
       <string>Enter path to query file...</string>
      </property>
     </widget>
     <widget class="QLabel" name="queryPathLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Path to Query File:</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="openQueryBuilderButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Launch Query Builder</string>
      </property>
     </widget>
     <zorder>applyQueryButton</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>collectionBrowseButton</zorder>
     <zorder>comboBox</zorder>
     <zorder>queryBrowseButton</zorder>
     <zorder>collectionPathLabel</zorder>
     <zorder>label_5</zorder>
     <zorder>collectionPathInput</zorder>
     <zorder>processButton</zorder>
     <zorder>queriesComboLabel</zorder>
     <zorder>queryLoadButton</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox</zorder>
     <zorder>queriesCombo</zorder>
     <zorder>queryPathInput</zorder>
     <zorder>queryPathLabel</zorder>
     <zorder>progressBar</zorder>
     <zorder>progressLabel</zorder>
     <zorder>openQueryBuilderButton</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Single Document Panel</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>LSA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lex Rank</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>More coming soon!!</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="filePathLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Path to File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Sentences:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fileBrowseButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>*Approximate</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinSentences">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="filePathInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="plainText">
       <string>C:/Users/zwelz3/Documents/GTRI_Projects/ECCT_EW_EMS/Market Research Sample/Hypersonic Weapons/Lockheed Martin tasked with developing hypersonic missiles.pdf</string>
      </property>
      <property name="placeholderText">
       <string>Enter path to file...</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinWords">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Summary Engine:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Key Words/Phrases:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="summarizeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Generate Results</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Results Panel</string>
     </attribute>
     <widget class="QPushButton" name="saveOutputButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>540</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save to file</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="outputText">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>411</width>
        <height>501</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nothing to see here...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Preview:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Matches</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Preview:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="outputMatches">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>411</width>
        <height>531</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nothing to see here...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLoadQL"/>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionSaveQL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoadQL">
   <property name="icon">
    <iconset>
     <normaloff>icons/open.png</normaloff>icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Load Query File</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset>
     <normaloff>icons/refresh.png</normaloff>icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionSaveAsQL">
   <property name="icon">
    <iconset>
     <normaloff>icons/save.png</normaloff>icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
